emuDCS/PeripheralCore/include/emu/pc/TMB.h

```diff
@@ -2095,6 +2209,28 @@ class TMB :  public VMEModule, public EMUjtag, public EmuLogger {
   //GTX link error count (full scale count is hex E0)
   inline int  GetReadGtxRxErrorCount(int cfebNum) { return read_gtx_rx_error_count_[cfebNum]; }
   
+  inline int GetReadTMBMMCMLockTime() { return read_tmb_mmcm_lock_time_;}
+  inline int GetReadTMBPowerUpTime() { return read_tmb_power_up_time_;}
+  inline int GetReadTMBLoadCfgTime() { return read_tmb_load_cfg_time_;}
+  inline int GetReadALCTPhaserLockTime() { return read_alct_phaser_lock_time_;}
+  inline int GetReadALCTLoadCfgTime() { return read_alct_load_cfg_time_;}
+  inline int GetReadGtxRstDoneTime() { return read_gtx_rst_done_time_;}
+  inline int GetReadGtxSyncDoneTime() { return read_gtx_sync_done_time_;}
+  //---------------------------------------------------------------------
+  // 0X15C ADR_V6_CFEB_BADBITS_CTRL: CFEB Bad Bits Control/Status (See Adr 0x122) (extra DCFEB Bad Bits on OTMB)
+  //---------------------------------------------------------------------
+  //!dcfeb_badbits_reset = 1 = Set the reset bit for the "badbits" marker for the last 2 CFEBs
+  void SetDCFEB56BadBitsReset(int dcfeb_badbits_reset);
+  int  GetDCFEB56BadBitsReset();
+  int  GetReadDCFEB56BadBitsReset();
+  //
+  //!dcfeb_badbits_block = 1 = Block channels which have been determined to be "badbits" by the firmware
+  void SetDCFEB56BadBitsBlock(int dcfeb_badbits_block);
```

Comment:
this is currently not used, but it seems like it should be just called inside the SetCFEBBadBitsBlock  (the same for the pieces below: reads and writes should go to both the 0..4 and then 5..6 for common cases
... should be on the list of incomplete features, I suppose